docs/kb/api-version-4/prospects.md

```diff
@@ -287,22 +285,23 @@ XML Example:
         <first_name>New first name</first_name>
         <last_name>New last name</last_name>
     </prospect>
-    <prospect identifier="some@email.com">
+    <prospect identifier="1337">
         <first_name>New first name</first_name>
         <last_name>New last name</last_name>
     </prospect>
-    <prospect identifier="some.other@email.com">
+    <prospect identifier="9">
         <first_name>New first name</first_name>
         <last_name>New last name</last_name>
     </prospect>
 </prospects>
 ```
 
-If using `batchCreate`, you'll need to provide a valid and unique email address for each prospect. The `batchUpdate` and `batchUpsert` endpoints allow the use of a prospect ID or prospect email address.
+If using `batchCreate`, you'll need to provide a valid and unique email address for each prospect. The `batchUpdate`  endpoints allow the use of prospect IDs and `batchUpsert` allows the use of prospect IDs and prospect Email Addresses.
```

Comment:
The unique constraint on email is no longer needed; that's why we quit keying on email in the input format.